src/Symfony/Component/VarDumper/Dumper/CliDumper.php

```diff
@@ -58,8 +58,8 @@ public function __construct($output = null, $charset = null, $flags = 0)
     {
         parent::__construct($output, $charset, $flags);
 
-        if ('\\' === DIRECTORY_SEPARATOR && false !== @getenv('ANSICON')) {
-            // Use only the base 16 xterm colors when using ANSICON
+        if ('\\' === DIRECTORY_SEPARATOR && 'ON' !== @getenv('ConEmuANSI') && 'xterm' !== @getenv('TERM')) {
+            // Use only the base 16 xterm colors when using ANSICON or standard Windows 10 CLI
```

Comment:
I'm not sure this change is right. Here, we detect if ANSICON is in use to fallback to 16 colors. I don't get why it needs any changes.